zcash_local_net/src/process.rs

```diff
@@ -0,0 +1,34 @@
+//! common behavior to processes
+
+use std::future::Future;
+
+use crate::{error::LaunchError, Process};
+
+/// Processes share some behavior.
+pub trait IsAProcess: Sized {
+    /// Process
+    const PROCESS: Process;
+
+    /// Indexer config struct
```

Comment:
A process may be a validator or an indexer 
```suggestion
    /// Config struct
```